### ISSUE TITLE
Germany Edition is missing from application.properties

### ISSUE THREAD

**Issue by sebg-mio42** [IHTSDO/snowstorm#576]
The newly released Germany Edition of SNOMED CT is currently missing from src/main/resources/application.properties

**Comment by kaicode** [comment]
Thanks for pointing that out!
I'll get that added to the develop branch to be merged into the next release candidate.
Does this look okay?
```
# (Format is codesystem.config.{CodeSystemShortName}={Name}|{DefaultModule}|{CountryCode}|{Owner} )
codesystem.config.SNOMEDCT-DE=Germany|11000274103|de|Bundesinstitut für Arzneimittel und Medizinprodukte, BfArM
```

**Comment by kaicode** [comment]
Internal ticket: [INFRA-12239](https://jira.ihtsdotools.org/browse/INFRA-12239)

**Comment by kaicode** [comment]
You are right. "German Edition" would be consistent with the other entries in the application.properites. This name will show up in the FHIR CodeSystem listing.

**Comment by kaicode** [comment]
Merged into RC 10.1.0.

**Comment by kaicode** [comment]
This is fixed in the release 10.1.0, out today - https://github.com/IHTSDO/snowstorm/releases/tag/10.1.0
